src/wfs/url.ts

```diff
@@ -51,6 +53,15 @@ export function generateGetFeatureUrl(
     const extentJoined = extent.join(',');
     newParams.BBOX = extentCrs ? `${extentJoined},${extentCrs}` : extentJoined;
   }
+  if (startIndex !== undefined) {
+    if (version >= '2.0.0') {
```

Comment:
I think you can remove this check altogether; many implementations (inc. GeoServer) will support startIndex even using lower versions of WFS, so adding it anyway cannot hurt. On the other side, since we now provide a `supportsStartIndex` method in the endpoint, it is up to the application code to handle the case where this might have no effect at all (e.g. disabling a field).

Reply:
Yes, sounds correct.
I have made the suggested updates